src/Owin.Security.Providers.VKontakte/VKontakteAuthenticationHandler.cs

```diff
@@ -95,6 +95,10 @@ protected override Task ApplyResponseChallengeAsync()
                 var user = await GetUser(response, accessToken);
 
                 var context = CreateAuthenticatedContext(user, accessToken, properties);
+    
+                // Email support. VK send it with access_token
+                context.Identity.AddClaim(new Claim(ClaimTypes.Email, response["email"].ToString(), XmlSchemaString,
```

Comment:
2 things...

1. Can this line be moved into the CreateAuthenticatedContext() function?
2. Can we instead get the email out of the response and set it to a property in `VKontakteAuthenticatedContext`, and then use that property in this line (where `response["email"].ToString()` is)